src/eureka/S3_data_reduction/straighten.py

```diff
@@ -35,7 +35,9 @@ def find_column_median_shifts(data, meta, m):
                    np.sum(data, axis=0))
 
     # Smooth CoM values to get rid of outliers
-    smooth_coms = smooth.medfilt(column_coms, meta.window_len)
+    smooth_coms = smooth.medfilt(column_coms, 11)
```

Comment:
Why force a window length of 11?

Reply:
We were using `meta.window_len` for multiple purposes and it became a problem.  In order to test `optspex.clean_median_flux()` with window_len = 1, I could not have `smooth.medfilt()` in straighten.py also using a window length of 1.  After some thought, I concluded that we will never need to change the smoothing window length, so I fixed it to 11.